components/profile/c_info_card.vue

```diff
@@ -26,7 +26,7 @@
               <v-type-divider type="Info"></v-type-divider>
             </v-flex>
             <v-flex xs12 md12 class="ma-0 pa-0">
-              <v-address-card :address="address"></v-address-card>
+              <v-address-card :address="address">{{userdata.address}}</v-address-card>
```

Comment:
Hello @MexinaD `v-address-card` is a custom component defined in [components](https://github.com/amalatechnologies/webapp/blob/master/components/profile/c_address_card.vue)  and registered to [c_cards](https://github.com/amalatechnologies/webapp/blob/master/static/js/c_card.js#L11) as child component of c_card_info with name `v-address-card`.

Since it is a custom component and child of another component, In vue we can only pass data to a child component by using [props](https://v2.vuejs.org/v2/guide/components-props.html) and not [Interpolation](https://vuejs.org/guide/essentials/template-syntax.html#text-interpolation)

The [address component](https://github.com/amalatechnologies/webapp/blob/master/components/profile/c_address_card.vue#L20) will expect address of type Object. So we should pass it as ` :address="ADDRESS_OBJECT"`